www/include/monitoring/downtime/template/AddDowntime.ihtml

```diff
@@ -124,12 +117,20 @@
         </tr>
         <tr class="list_two">
             <td class="FormRowField">{$form.start.label}</td>
-            <td class="FormRowValue">{$form.start.html}&nbsp;{$form.alternativeDateStart.html}&nbsp;{$form.start_time.html}
+            <td class="FormRowValue">{$form.start.html}&nbsp;
+            {if isset($form.alternativeDateStart)}
+                {$form.alternativeDateStart.html}
+            {/if}
+            &nbsp;{$form.start_time.html}
             </td>
         </tr>
         <tr class="list_one">
             <td class="FormRowField">{$form.end.label}</td>
-            <td class="FormRowValue">{$form.end.html}&nbsp;{$form.alternativeDateEnd.html}&nbsp;{$form.end_time.html}
+            <td class="FormRowValue">{$form.end.html}&nbsp;
+            {if isset($form.alternativeDateEnd)}
+                {$form.alternativeDateEnd.html}
```

Comment:
this one is mandatory to be able to localize the date.
Shouldn't you set it if not set ?

Reply:
I removed totally this check, cause alternative dates are already set by `$form.hidden`